pkgs/development/python-modules/aplpy/default.nix

```diff
@@ -27,13 +27,24 @@ buildPythonPackage rec {
     sha256 = "239f3d83635ca4251536aeb577df7c60df77fc4d658097b92094719739aec3f3";
   };
 
-  patches = [ (fetchpatch {
+  patches = [
+    (fetchpatch {
       # Can be removed in next release after 2.0.3
       url = "https://github.com/aplpy/aplpy/pull/448.patch";
       sha256 = "1pnzh7ykjc8hwahzbzyryrzv5a8fddgd1bmzbhagkrn6lmvhhpvq";
       excludes = [ "tox.ini" "azure-pipelines.yml" ".circleci/config.yml" "MANIFEST.in" ".gitignore"
        "setup.cfg" "appveyor.yml" "readthedocs.yml" "CHANGES.rst" ".gitmodules" ".travis.yml" "astropy_helpers" ];
     })
+    # Fix for matplotlib >= 3.4 (https://github.com/aplpy/aplpy/pull/466)
+    # Note: because of a security thing, github will refuse to serve this patch from the
+    # "normal" location
+    # (https://github.com/aplpy/aplpy/commit/56c1cc694fdea69e7da8506d3212c4495adb0ca5.patch)
+    # due to the fact that it contains the PDF magic bytes, but it will serve it from
+    # this githubusercontent domain.
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/aplpy/aplpy/pull/466.patch";
```

Comment:
Could you point at the actual commit id underneath this url? The contents could change if you're asked to force-push before it gets merged.

Reply:
I am not actually sure how to a url that points to the commit id for this patch-diff.githubusercontent.com domain. I could just copy the patch into our tree.

Reply:
A little bit of guessing... https://patch-diff.githubusercontent.com/raw/aplpy/aplpy/pull/466/commit/56c1cc694fdea69e7da8506d3212c4495adb0ca5.patch seems to WFM